.github/workflows/leancode_hooks-test.yml

```
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - version: 3.0.5
+          - version: 3.10.6
```

Let's also test the newest version

```suggestion
          - version: 3.10.6
          - version: 3.13.5
```